### ISSUE TITLE
Can't open attributes on the root group

### ISSUE THREAD

**Issue by ChaosCA** [gonum/hdf5#39]
### What are you trying to do?
View an attribute on the root path of the HDF5 File "/"

### What did you do?
```
		f, err := hdf5.OpenFile(event.Name(), hdf5.F_ACC_RDONLY)
		if err != nil {
			log.Fatalln("Panic, could not open file")
		}

		d, err := f.OpenGroup("/")
		if err != nil {
			log.Fatalln("Panic, could not open dataset")
			return
		}

		attr, err := d.OpenAttribute("file_version")
```
OpenAttributes on a group doesn't seem to work. You can however call CreateAttribute, just not read from it.

### What did you expect to happen?
The ability to open an attribute like on a dataset.

### What version of Go, Gonum, Gonum/netlib and libhdf5 are you using?

go version go1.10.2 linux/amd64

### Does this issue reproduce with the current master?
Unknown

**Comment by TacoVox** [comment]
Hi @ChaosCA 

Is it even necessary to open the group your particular example? Can you try to call
`f.OpenAttribute("file_version")` and remove the group related code instead?

Cheers!

**Comment by ChaosCA** [comment]
I don't believe that method exists @TacoVox 
I can't OpenAttributes against a file handler.

**Comment by delaneyj** [comment]
@ChaosCA did you ever find a solution or workaround?

**Comment by ChaosCA** [comment]
Nope, you could directly call the C API though. I believe that is what I ended up doing.
Or just defer work to the excellent Python API for HDF5.

For C, you'd open the attribute, similar to how it's done here: https://github.com/gonum/hdf5/blob/master/h5a_attribute.go#L35

And using this function: https://support.hdfgroup.org/HDF5/doc/RM/RM_H5A.html#Annot-Open

And then you'd read the data from the attribute.
